packages/webpack-plugin/src/index.js

```diff
@@ -19,13 +19,16 @@ class LoadablePlugin {
 
   handleEmit = compilation => {
     const stats = compilation.getStats().toJson({
-      hash: true,
-      publicPath: true,
+      all: false,
       assets: true,
       chunks: true,
+      chunkGroups: true,
+      errorDetails: false,
```

Comment:
Is there a reason to keep these `false` options provided? Or does `all: false` set that as a default for anything not explicitly provided?

Reply:
looking at https://github.com/webpack/webpack/blob/afc9b2fcf9bef0831640b3ebb02b73068ba18e17/lib/stats/DefaultStatsPresetPlugin.js#L132 and https://webpack.js.org/configuration/stats/#statsall, it doesn't seem so since `all: false` will basically default anything not set to `false`.

i opted for leaving these as-is since i was trying to make a minimal change here but happy to remove the unnecessary false options here

Reply:
IMO it'd be clearer to just totally omit anything that isn't `true`, since the `all: false` covers all that. But I'll defer to the repo maintainers, I'm just taking a look since this PR will solve my issue too (#709) 😄